src/casanovoutils/summarize_mgf.py

```
@@ -0,0 +1,1247 @@
+                        for mod in (mods or []):
+                            mod_counts[(residue, str(mod))] += 1
+                    for mod in (props.get("n_term") or []):
+                        mod_counts[("N-term", str(mod))] += 1
+                    for mod in (props.get("c_term") or []):
+                        mod_counts[("C-term", str(mod))] += 1
+                except Exception:
+                    pass
+
+            n_with_charge = sum(charge_counts.values())
+            print(f"  {total_spectra:,} spectra loaded.", file=sys.stderr)
+            print(f"Computing charge distribution ...", file=sys.stderr)
```

Ruff (0.15.2)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)